src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -651,6 +651,8 @@ private function writeProperty($zval, $property, $value)
             // fatal error.
 
             $object->$property = $value;
+        } elseif (self::ACCESS_TYPE_NOT_FOUND === $access[self::ACCESS_TYPE]) {
```

Comment:
@bramtweedegolf can you please move this elseif one step down, to put all throwing cases at the end?